.eslintignore

```diff
@@ -0,0 +1,3 @@
+/specs/samples/**
+/coverage/**
+/node_modules/**
```

Comment:
Why the missing EOL? Also, eslint **always** ignores node_modules, there's no point in listing it explicitly.